docs/guides/element-builder/tips.md

```diff
@@ -0,0 +1,66 @@
+---
+heading: Element Builder
+seo: Tips | Element Builder | Cloud Elements API Docs
+title: Tips
+description: Element Builder API Documentation.
+layout: sidebarleft
+apis: API Docs
+platform: elementsbuilder
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 50
+sitemap: false
+---
+# Tips
+
+Tips is a section to find all of your tips and tricks for building high quality elements! *More to come!*
+
+Have a tech tip suggestion? Please contact [Support](mailto:support@cloud-elements.com).
+
+### **Pagination Max**
+
+When building an element in Element Builder, you set up a max pagination, (*number 10 pictured below*) for your element. Cloud Elements uses this to discern what to send to the vendor as a default in the __pageSize__ parameter when making GET list request.
```

Comment:
Makes sense 👍 

I was just borrowing the Twitter image from the 'Build an Element from UI' page, but I can add a different one. 

No, it's not a `GET /lists` request per se. It's a GET request that returns a list response (_which could be a `GET /lists` request, or `contacts`, `users`, etc._). I couldn't really think of the best phrasing for that. What would you recommend there?